### PR TITLE
#125 - Introduced unittest helper class

### PR DESCRIPTION
**Pull request by kvanhijf**
Removed some obsolete arakoon files

### CODE REVIEW

File: ovs/dal/hybrids/storagedriver.py [hunk 1]
```diff
@@ -24,14 +24,14 @@
...
 
-logger = LogHandler.get('dal', name='hybrid')
-
 
 class StorageDriver(DataObject):
     """
     The StorageDriver class represents a Storage Driver. A Storage Driver is an application
     on a Storage Router to which the vDisks connect. The Storage Driver is the gateway to the Storage Backend.
     """
+    _logger = LogHandler.get('dal', name='hybrid')
```
Comment (khenderick) [inline ovs/dal/hybrids/storagedriver.py:33]: I wonder whether a hybrid can use the logger available in the `DataObject`. If not, try by - in `DataObject` - assign the logger to sth like `self._logger`, and then I think you can just use `self._logger` in the hybrids as well.

File: ovs/dal/performance_test.py [hunk 2]
```diff
@@ -28,17 +27,20 @@
...
 
 
-class LotsOfObjects(TestCase):
+class LotsOfObjects(object):
     """
     Executes a performance test by working with a large set of objects
     """
+    def __init__(self):
+        """
+        Dummy init method
```
Comment (khenderick) [inline ovs/dal/performance_test.py]: It's not dummy anymore

File: ovs/dal/hybrids/storagedriver.py (hunk 1 above)
Comment (kvanhijf) [inline ovs/dal/hybrids/storagedriver.py:33]: Keeping it as it is after discussing the issue

File: ovs/dal/performance_test.py (hunk 2 above)
Comment (kvanhijf) [inline ovs/dal/performance_test.py]: Fixed

**Pull request merged**